src/gc-opt/gc-opt.cpp

```diff
@@ -46,13 +52,23 @@ int main(int argc, char *argv[]) {
 #endif
   mlir::registerAllPasses();
   mlir::gc::registerCPUPipeline();
+  mlir::gc::registerGPUPipeline();
```

Comment:
```suggestion
#ifdef GC_USE_GPU
  mlir::gc::registerGPUPipeline();
#endif
```

Reply:
What is this for?

Reply:
We have the build option for enabling/disabling GPU.  It could make sense if GPU is disabled.